### PR TITLE
[RFR] Url vers requete

### PR DESCRIPTION
**Pull request by Leo-Lemaitre**

### CODE REVIEW

File: src/Filetype.js [hunk 1]
```diff
@@ -33,9 +41,33 @@ export default class Filetype extends React.Component {
...
+    getAlert() {
+    alert('getAlert from Child');
+  }
+
+
+    check(event){
+        console.log("filtype "+this.props.filetype);
+        console.log("Les formats",this.props.formats);
+        if(this.props.disabled===false){
+      this.setState({
```
Comment (parmentf) [inline src/Filetype.js]: Strange indentation.
Please use spaces to indent (this is environment-independant).

File: src/Filetype.js [hunk 2]
```diff
@@ -33,9 +41,33 @@ export default class Filetype extends React.Component {
...
+          console.log("TEST",this.state.count)
+        c.check(new Event("format"));
+      });
+    }
+  }
+
     handleInputChange(event) {
         const target = event.target;
-        const value = target.type === 'checkbox' ? target.checked : target.value;
+        const  value = target.type === 'checkbox' ? target.checked : target.value;
```
Comment (parmentf) [inline src/Filetype.js]: unintended?

File: src/Filetype.js [hunk 3]
```diff
@@ -33,9 +41,33 @@ export default class Filetype extends React.Component {
...
     }
 
+    getAlert() {
+    alert('getAlert from Child');
+  }
+
+
+    check(event){
+        console.log("filtype "+this.props.filetype);
+        console.log("Les formats",this.props.formats);
```
Comment (parmentf) [inline src/Filetype.js]: These `console.log` should be useful only during debugging.
And thus, not present in a Request For Review, which aims to be merged into master branch.

There are still `console.log`s in other files too.

File: src/Form.js [hunk 4]
```diff
@@ -14,11 +14,40 @@ export default class Form extends React.Component {
 
     constructor(props) {
         super(props);
+
+        const url=document.location.href;
+        console.log("AAAAAAAAAAAA",this.url);
+        //TRouver meilleure condition
+        if(url.indexOf("q=")>=0){
```
Comment (touv) [inline src/Form.js]: j'utilsierai https://www.npmjs.com/package/qs pour analyser la querystring

**Pull request merged**